app/views/shared/check_answers/_income_summary.html.erb

```diff
@@ -11,13 +11,26 @@
     </div>
 </div>
 
-<dl class="govuk-summary-list govuk-!-margin-bottom-9" data-check-your-answers-section="<%= income_type.parameterize %>">
+<% if @legal_aid_application.using_enhanced_bank_upload? %>
+  <dl class="govuk-summary-list govuk-!-margin-bottom-9" data-check-your-answers-section="<%= income_type.parameterize %>">
```

Comment:
perhaps the income_type can be passed to the partial and this would resolve the bug that the <dl> is defined outside the partial with the <dt> and <dd>.